torch/csrc/jit/codegen/cuda/lower_fused_reduction.cpp

```diff
@@ -60,8 +60,8 @@ class FusionInspector : private IterVisitor {
   using IterVisitor::handle;
 
   void handle(ReductionOp* rop) final {
-    /// If it's a grid reduction, keep track of tensors that depend on
-    /// this reduction.
+    // If it's a grid reduction, keep track of tensors that depend on
```

Comment:
Changes in this file are not related to the outer grouped grid welford, but just making the pattern matching of reduction and broadcast stricter.